src/MultiContainerTrait.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+
+namespace atk4\core;
+
+
+/**
+ * This trait makes it possible for you to add child objects
+ * into your object, but unlike "ContainerTrait" you can use
+ * multiple collections stored as different array properties.
+ *
+ * This class does not offer automatic naming, so if you try
+ * to add another element with same name, it will result in
+ * exception.
+ */
+trait MultiContainerTrait
+{
+
+    /**
+     * Use this method trait like this:
+     *
+     * function addField($name, $definition) {
+     *     $field = $this->factory($definition, [], '\atk4\data\Field');
+     *
+     *     return $this->_addIntoCollection($name, $field, 'fields');
+     * }
+     *
+     * @param $name string Name that can be used to reference object
```

Comment:
order should be @param, then type and then name of argument

Reply:
fixed.